source/includes/usage-examples/code-snippets/findOne.go

```diff
@@ -35,16 +35,18 @@ func main() {
 
 	// begin findOne
 	coll := client.Database("sample_mflix").Collection("movies")
+	
 	var result bson.M
 	err = coll.FindOne(context.TODO(), bson.D{{"title", "The Room"}}).Decode(&result)
-	// end findOne
 	if err != nil {
 		if err == mongo.ErrNoDocuments {
 			// This error means your query did not match any documents.
 			return
```

Comment:
The code sample currently prints a result whether a runtime error or a result, and nothing if it doesn't match any documents. Would it be helpful to print the query didn't match any documents in this conditional block?

Reply:
The example in the API Docs for [FindOne()](https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.2/mongo#Collection.FindOne) only returns, like this example currently does. I noticed that when I create code snippets that differ from the API examples, the engineers usually ask me to fix mine to match.